src/components/Header/AppHeader.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import styles from "./AppHeader.module.css";
+
+interface AppHeaderProps {
+  cardsAmount: number;
+}
+
+export const AppHeader = ({cardsAmount=1}:AppHeaderProps) => {
+ 
+  return (
+    <header className={styles.header}>
+      <div className={styles.header__left}>
+        <div>
+          <svg
+            width="34"
```

Comment:
**nitpick:** I strongly suggest to extract this SVG, as separate React component to keep it simple, and readable